### PR TITLE
Added new WidgetSize

### PR DESCRIPTION
**Pull request by GuidoZam**
| Q               | A
| --------------- | ---
| Bug fix?        | [ ]
| New feature?    | [x]
| New sample?      | [ ]
| Related issues?  | fixes #X, partially #Y, mentioned in #Z

#### What's in this Pull Request?

Added a new WidgetSize to expand the widget width to four places inside the Dashboard control.

### CODE REVIEW

**Review by michaelmaillot — CHANGES_REQUESTED** [review]
Hi @GuidoZam,

Thanks for this improvment! I've added few things to update then we'll be good to go 🙂

File: src/controls/dashboard/widget/IWidget.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ export enum WidgetSize {
   Single = "single",
   Double = "double",
   Triple = "triple",
+  Quadruple = "quadruple",
```
[the remark this answers is not in the document]
Comment (GuidoZam) [reply src/controls/dashboard/widget/IWidget.ts:30]: Updated in the next commit.

File: src/controls/dashboard/widget/Widget.tsx [hunk 2]
```diff
@@ -28,6 +28,9 @@ export const Widget = ({
     if (widget.size === WidgetSize.Triple) {
         cardStyle.gridColumnEnd = "span 3";
     }
+    if (widget.size === WidgetSize.Quadruple) {
```
[the remark this answers is not in the document]
Comment (GuidoZam) [reply src/controls/dashboard/widget/Widget.tsx:31]: Updated in the next commit.

**Comment by GuidoZam** [comment]
Hi @michaelmaillot, I've resolved your change requests 😊

**Comment by michaelmaillot** [comment]
Merged manually, thank you!

**Pull request merged**